newdle/client/src/components/creation/CreationSuccessPage.js

```diff
@@ -38,7 +38,19 @@ export default function CreationSuccessPage() {
             invite:
           </p>
         )}
-        <div className={styles['newdle-link']}>{newdle.url}</div>
+        <Input
+          className={styles['newdle-link']}
+          readOnly={true}
+          value={newdle.url}
+          action={{icon: 'copy'}}
+        >
+          <input />
```

Comment:
that looks wrong, do we need that?!

Reply:
Yeah, I think that is not required. Going to remove it